templates/simple-gmap-output.tpl.php

```diff
@@ -48,4 +53,4 @@
 ?>
 <p class="simple-gmap-address"><?php print $address_text; ?></p>
 <?php
-}
+}
```

Comment:
Can you add an extra newline at the end of this file please?

Reply:
Done